src/inspector_agent.h

```diff
@@ -23,6 +25,28 @@ namespace inspector {
 
 class AgentImpl;
 
+class ContextInfo {
+ public:
+  explicit ContextInfo(v8::Local<v8::Context> context, int groupId,
+                       const char* name)
+                       : groupId_(groupId),
+                       name_(name) {
+                       context_.Reset(context->GetIsolate(), context);
+  }
+  ~ContextInfo() {
+    context_.Reset();
+  }
+  inline v8::Local<v8::Context> context(v8::Isolate* isolate) const {
+    return context_.Get(isolate);
+  }
+  int groupId() const { return groupId_; }
+  const char* name() const { return name_; }
+ private:
+  v8::Persistent<v8::Context> context_;
+  int groupId_;
```

Comment:
Can you snake_case it to `group_id_`?  If it never changes, consider making it const (same for `name_`.)